reana_server/rest/analyses.py

```diff
@@ -488,3 +491,94 @@ def set_analysis_status(analysis_id):  # noqa
         return jsonify({"message": str(e)}), 404
     except Exception as e:
         return jsonify({"message": str(e)}), 500
+
+
+@blueprint.route(
+    '/analyses/<analysis_id>/workspace/outputs/<path:file_name>',
+    methods=['GET'])
+def get_analysis_outputs_file(analysis_id, file_name):  # noqa
+    r"""Get analysis status.
+
+    ---
+    get:
+      summary: Returns the requested file.
+      description: >-
+        This resource is expecting a workflow UUID and a file name to return
+        its content.
+      operationId: get_analysis_outputs_file
+      produces:
+        - multipart/form-data
+      parameters:
+        - name: organization
+          in: query
+          description: Required. Organization which the analysis belongs to.
+          required: true
+          type: string
+        - name: user
+          in: query
+          description: Required. UUID of analysis owner.
+          required: true
+          type: string
+        - name: workflow_id
+          in: path
+          description: Required. analysis UUID.
+          required: true
+          type: string
+        - name: file_name
+          in: path
+          description: Required. Name (or path) of the file to be downloaded.
+          required: true
+          type: string
+      responses:
+        200:
+          description: >-
+            Requests succeeded. The file has been downloaded.
+          schema:
+            type: file
+        400:
+          description: >-
+            Request failed. The incoming data specification seems malformed.
+        404:
+          description: >-
+            Request failed. `file_name` does not exist .
+          examples:
+            application/json:
+              {
+                "message": "input.csv does not exist"
+              }
+        500:
+          description: >-
+            Request failed. Internal controller error.
+          examples:
+            application/json:
+              {
+                "message": "Either organization or user doesn't exist."
+              }
+    """
+    try:
+        user = request.args['user'],
+        organization = request.args['organization'],
+        workflow_id = analysis_id
+
+        response, http_response = rwc_api_client.api.get_workflow_outputs_file(
+            user=request.args['user'],
+            organization=request.args['organization'],
+            workflow_id=analysis_id,
+            file_name=file_name).result()
+
+        return send_file(
+            io.BytesIO(http_response.raw_bytes),
+            attachment_filename=file_name,
+            mimetype='multipart/form-data'), 200
+    except (KeyError, HTTPBadRequest) as e:
+        return jsonify({"message": str(e)}), 400
+    except HTTPError as e:
+        # we have this option, to blindly mirror responses from RWC or
```

Comment:
@tiborsimko @hjhsalo any opinion here? I would rather prefer to explicitly specify exceptions so you can directly check and update the `docstring`. If we just redirect what comes from RWC we would have to check RWC `openapi` spec to write the `docstring` (which is likely to not going to happen)

Reply:
I'd probably favour explicit over implicit...